modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2TeamMembersRemoveEditRoles.testcase

```diff
@@ -0,0 +1,321 @@
+	@priority = "5"
+	test CanViewIfTheUserIsRemoved {
+		task ("Adding a user to the project") {
+			JSONUser.addUserWithRole(
+				roleTitle = "Power User",
+				userEmailAddress = "test.user@liferay.com",
+				userFirstName = "Test",
+				userLastName = "User",
+				userScreenName = "Test");
+
+			Accounts.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Test Account DXP");
+
+			Accounts.assignUsers(
+				accountName = "Test Account DXP",
+				assigneeName = "Test User");
+		}
+
+		task ("Change the role of the current user in the project") {
+			Accounts.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and login to okta") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Close the confirmation modal") {
+			CP2Utils.openActionMenu(
+				menuItem = "Remove",
+				openUser = "Test User");
+
+			Button.click(button = "Remove");
+		}
+
+		task ("Assert the success message when submitting the form") {
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+
+		task ("Delete user created") {
+			JSONUser.deleteUserByEmailAddress(userEmailAddress = "test.user@liferay.com");
+		}
+	}
+
+	@description = "Automation ID: LPS-150012 | Test Summary: Verify that after the user clicks Edit a Save button is available"
+	@priority
+	test CanViewSaveButton {
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and login to okta") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Open the action menu and click the Edit option") {
+			CP2Utils.openActionMenu(
+				menuItem = "Edit",
+				openUser = "Test Test");
+		}
+
+		task ("Change the guest user role and save") {
+			Button.click(button = "Save");
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
```

Comment:
Besides only guaranteeing the button is available, can we test also if the "save" button will indeed confirm the change? For example, change something and verify that when the user hits "Save" the change will persist.

Also, this could be applied for the Cancel. You change something, clicks Cancel and verify nothing was modified.

Reply:
Done.